lib/kiwi_config.rb

```diff
@@ -124,7 +125,8 @@ def inject_extracted_files(output_location)
       FileUtils.mkdir_p(destination, mode: 01777)
       filter = "unmanaged_files_#{@name}_excludes"
 
-      @system_description.unmanaged_files.export_files_as_tarballs(destination)
+      @system_description.unmanaged_files.file_access.
+        export_files_as_tarballs(destination)
```

Comment:
I think I'm not following. The separation would be have two different modules, just like the two classes we have right now. 
The encapsulation by name space (I think you're referring to the `.file_access` part?) is what I don't like, I think the file scopes could know how to write their files on their own.